src/Benchmarks.ServerJob/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0-*",
+  "description": "Class to transfer data to/from the benchmark server.",
```

Comment:
I think instead of dealing with the boilerplate to create a project for the ServerJob/ClientJob transfer classes, it would be easier just to have some shared .cs file [like this](https://github.com/aspnet/EntityFramework/blob/c28771c3244a7c19d42fb58d500c3b455942ac71/src/Microsoft.EntityFrameworkCore/project.json#L19).